Lib/test/support/__init__.py

```diff
@@ -505,10 +505,10 @@ def requires_lzma(reason='requires lzma'):
 
 def has_no_debug_ranges():
     try:
-        import _testinternalcapi
+        import _testcapi
     except ImportError:
         raise unittest.SkipTest("_testinternalcapi required")
-    config = _testinternalcapi.get_config()
+    return not _testcapi.config_get('code_debug_ranges')
     return not bool(config['code_debug_ranges'])
```

Comment:
The return statement added on Line 511 is shadowing this return.

Reply:
Ooops, well spotted, I created https://github.com/python/cpython/pull/137379 to remove the dead code.